docs/community/roadmap.md

```diff
@@ -96,49 +96,72 @@ Investigate to cooperate with argo to support job dependencies.
 * Owner: @peiniliu
 
 ## v1.4 (To be updated)
+### Support NUMA-Awareness scheduling in Volcano
+* Description: Support NUMA-Awareness scheduling in Volcano.
+* Priority: High
+* Issue: https://github.com/volcano-sh/volcano/issues/1707
+* Owner: @huone1 @william-wang
+
+### Support multi-scheduler by admission controller
+* Description: Use default scheduler for system daemon and Volcano scheduler for biz workload.
+* Priority: High
+* Issue: https://github.com/volcano-sh/volcano/issues/1322
+* Owner: @huone1 @william-wang
+
+### Support scheduling with proportion of resources
+* Description: Add scheduling policy with proportion of resources.
+* Priority: Middle
+* Issue: https://github.com/volcano-sh/volcano/issues/1368
+* Owner: @king-jingxiang
+
+### Enhance the resource comparison functions for various of scenarios
+* Description: Improve the Fundamental comparison functions
+* Priority: Middle
+* Issue: https://github.com/volcano-sh/volcano/issues/1525
+* Owner: @Thor-wl
+
+### System Stability Enhancement
+* Description: Add UT/E2E to cover more scenarios and add basic stress test.
+* Priority: Middle
+* Issue: https://github.com/volcano-sh/volcano/issues/1284
+* Owner: @rudeigerc
+
+## v1.5 (To be updated)
 ### Support Hierarchy Queue
 * Description: Support Hierarchy Queue algorithm.
```

Comment:
Please add more information such alpha version, which basic functions to provide. THX